src/index.js

```diff
@@ -83,10 +84,20 @@ class App extends React.Component {
     this.setState({ [event.target.name]: event.target.value });
   }
 
+  handleExportFormatChange = (event) => {
+    console.log(event.target.name, event.target.value)
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  exportTranscript = (exportFormat) => {
+    const {data, ext} = this.refs.transcriptEditor.exportData(this.state.exportFormat);
+    this.download(data, `${ this.state.mediaUrl } .${ext}`);
+  }
+
   getEditorContent = () => {
-    const tmpEditorsContnet = this.refs.transcriptEditor.getEditorContent(this.state.sttType);
+    const tmpEditorsContent = this.refs.transcriptEditor.getEditorContent(this.state.sttType);
 
-    this.download(JSON.stringify(tmpEditorsContnet, null, 2), `${ this.state.mediaUrl } .json`)
+    this.download(JSON.stringify(tmpEditorsContent, null, 2), `${ this.state.mediaUrl } .json`)
```

Comment:
I'd remove the space between the file name and the file extension

```suggestion
    this.download(JSON.stringify(tmpEditorsContent, null, 2), `${ this.state.mediaUrl }.json`)
```

[btw using the beta -  suggest a specific change to the line](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/)